docs/user_guide/customizing.rst

```diff
@@ -65,37 +67,55 @@ For a complete list of the theme variables that you may override, see the
 Replacing/Removing Fonts
 ========================
 
-The theme contains custom web fonts, in several formats, for different purposes:
+The theme includes the `FontAwesome 5 Free <https://fontawesome.com/icons?m=free>`__
+icon font (the ``.fa, .far, .fas`` styles, which are used for
+:ref:`icon links <icon-links>` and admonitions). 
+This is the only `vendored` font, and otherwise the theme by default relies on
+available system fonts for normal body text and headers.
+
+.. Attention::
+
+    Previously-included fonts like `Lato` have been removed, preferring
+    the most common default system fonts of the reader's computer. This provides
+    both better performance, and better script/glyph coverage than custom fonts,
+    and is recommended in most cases.
+
+The default body and header fonts can be changed as follows:
```

Comment:
@bollwyvl if you could take a quick look at this section (the text below here), that would be great